terraform/zones/zone.library.ucsb.edu.tf

```
@@ -86,6 +86,14 @@ zone_id = local.library-zone_id
   records = ["lb-haproxy-legacy-001.library.ucsb.edu."]
 }
 
+resource "aws_route53_record" "victorledgers-poc-library-ucsb-edu-CNAME" {
+zone_id = local.library-zone_id
+  name    = "victorledgers-poc.library.ucsb.edu."
+  type    = "CNAME"
+  ttl     = "360"
+  records = ["."]
```

[](url)
```suggestion
  records = ["d2isjfozjvdahq.cloudfront.net."]
```
i forgot to wait for the cf fqdn